doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-        while (it) {
-            kaa_topic_t \*topic = (kaa_topic_t \*)kaa_list_get_data(it);
-            printf("Topic: id '%lu', name: %s\n", topic->id, topic->name);
-            it = kaa_list_next(it);
-        }
-    }
-
-    int main()
-    {
-        printf("Kaa SDK client started\n");
-
-        kaa_error_t error_code = kaa_client_create(&kaa_client, NULL);
-        KAA_DEMO_RETURN_IF_ERROR(error_code, "Failed create Kaa client");
-
-        kaa_topic_listener_t topic_listener = { &on_topics_received, kaa_client };
-        kaa_notification_listener_t notification_listener = { &on_notification, kaa_client };
-
-        uint32_t topic_listener_id = 0;
-        uint32_t notification_listener_id = 0;
-
-        error_code = kaa_add_topic_list_listener(kaa_client_get_context(kaa_client)->notification_manager
-                                           , &topic_listener
-                                           , &topic_listener_id);
-        KAA_DEMO_RETURN_IF_ERROR(error_code, "Failed add topic listener");
-
-        error_code = kaa_add_notification_listener(kaa_client_get_context(kaa_client)->notification_manager
-                                             , &notification_listener
-                                             , &notification_listener_id);
-        KAA_DEMO_RETURN_IF_ERROR(error_code, "Failed add notification listener");
-
-        error_code = kaa_client_start(kaa_client, NULL, NULL, 0);
-        KAA_DEMO_RETURN_IF_ERROR(error_code, "Failed to start Kaa main loop");
-
-        kaa_client_destroy(kaa_client);
-
-        return error_code;
-    }
-```
-
-</div><div id="Objectve-C" class="tab-pane fade" markdown="1">
+## Avro UI
 
-```Objective-C
-    #import "ViewController.h"
-    #import <Kaa/Kaa.h>
+<!-- TODO: update link on Avro UI guide -->
+You can access Avro UI by clicking **Avro UI sandbox console** at the top of the window. 
+Refer to the [Avro UI guide](#) for working instructions.
 
-    #define TAG @"NotificationSystemTestApp >>>"
+## Configuration
 
-    @interface ViewController () <KaaClientStateDelegate, NotificationTopicListDelegate, NotificationDelegate, ProfileContainer>
+### Outgoing mail settings
 
-    @property (nonatomic, strong) id<KaaClient> kaaClient;
+Outgoing mail settings are used to send emails to newly created users with the information about their passwords, as well as other notifications. 
+By default, outgoing mail settings are not configured for Admin UI. 
+To target Admin UI to your SMTP server refer to the [Admin UI guide]({{root_url}}Administration-guide/Tenants-and-applications-management).
 
-    @end
+### Networking
 
-    @implementation ViewController
+By default, Kaa Sandbox components are accessible from a host machine only. But if you want to share Kaa Sandbox in the local network you need to reconfigure the network interface for this virtual machine in [Bridge mode](https://www.virtualbox.org/manual/ch06.html#network_bridged). 
```

Comment:
s/But if/If
s/network you/network, you
s/this virtual machine/the virtual machine